### DISCUSSION TITLE
Setting AUTH_LOCAL_DEFAULT_ROLE_ID in .env doesn't assign any role to user created through API

### DISCUSSION THREAD

**Discussion by adanielyan** [directus/directus#15067] [Archive]
### Describe the Bug

I set a ```AUTH_LOCAL_DEFAULT_ROLE_ID``` to an ID of existing custom role in my .env file (restarted the directus after setting it). However when I create users through GraphQL API their role field in directus remains null (I didn't try creating a user through REST API). Tried ```AUTH_DEFAULT_DEFAULT_ROLE_ID``` with the same result.

Is this a bug or am I doing something wrong?

### To Reproduce

1. Create a custom role
2. Edit .env file to add ```AUTH_LOCAL_DEFAULT_ROLE_ID="<NEWLY_CREATED_ROLE_ID>"```
3. Created a user through GraphQL API

**Comment by azrikahar** [comment]
`AUTH_<PROVIDER>_DEFAULT_ROLE_ID` does not work/apply for the `local` auth provider, which is why it's documented on every other provider _except_ `local`: https://docs.directus.io/self-hosted/config-options.html#authentication

If you wish to set a default role for a new user such as via a public role for example, you can try setting the Field Presets to configure a default role:

![image](https://user-images.githubusercontent.com/42867097/184567742-b32c9028-ab3e-41d9-8f7e-41b96a002f3e.png)

**Reply by mojerro** [reply]
hi, I cannot find the presets page, is it inside here?
![image](https://user-images.githubusercontent.com/15071824/187586340-7ae618e3-6697-4f1b-8175-7b2dffca599b.png)

**Reply by mojerro** [reply]
I find it, it's in roles~

**Comment by adanielyan** [comment]
Thank you, that worked!

Ironically, I implemented this same solution on another instance of directus a few months ago. I need to start taking notes!
